app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+
+  def create
+    @rent = Rent.create(rent_create_params)
+    render json: @rent, each_serializer: RentSerializer
+  end
+
+  def index
+    render_paginated Rent, each_serializer: RentSerializer
```

Comment:
Let's think of something. The rent belongs to a user. In this endpoint, you are returning all the rents. If you have a rent, do you want that anybody that is authenticated with your API sees all your rents?